### DISCUSSION TITLE
guys im having some issue tapo apps

### DISCUSSION THREAD

**Discussion by williamvyong** [microsoft/WSA#87] [General]
hi guys maybe i having the wrong place to discuss 
im installing tapo apps for my cctv 
but i dont know everytime open full screen on the tapo apps 
last not more then 2 minute and start pop up notification that telling me need to refresh 
or close it

**Comment by nieubank** [comment]
Can you file a bug on the Issues tab, if you could link to a logcat and provide a screen shot of what you're seeing that would be great.
